src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -105,6 +130,83 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     setAnchorEl(null);
   };
 
+  const userToAutocompleteOption = (user: User): { label: string; id: string } => {
+    return { label: `${fullNamePipe(user)} (${user.email})`, id: user.userId.toString() };
+  };
+
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) return;
+    try {
+      await requestCRReview.mutateAsync({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  const isRequestDisabled = changeRequest.submitter.userId !== currentUser.userId;
+
+  const openedACtionsDropdown = (
+    <div>
+      <NERButton
```

Comment:
this is the same thing as the component below this. I would just add it to that component and then only show it if the submitter is the current user

Reply:
and if it isnt reviewed yet

Reply:
Oop I also want to change the confirm text to be request review